src/pages/AddHouse/AddHouseForm.jsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useRef, useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import createHouses from '../../redux/actions/House/createHouses';
+
+function AddHouseForm() {
```

Comment:
- [ ] The functionality for adding houses is here, but I couldn't find any button or any way to add a house.  If the button is there, kindly make it bigger or more obvious.

If the "add house" button is behind an admin account, kindly share the credentials on the PR or in the review request(this is private) so people testing your app are able to test this functionality.

That way, your users will be able to add houses easily. 💪